### DISCUSSION TITLE
Type error when using useHydrateAtoms

### DISCUSSION THREAD

**Discussion by JuanJoEstrada** [pmndrs/jotai#2323] [Bug report]
I read the [docs](https://jotai.org/docs/extensions/query#referencing-the-same-instance-of-query-client-in-your-project) for incremental adoption of atomWithQuery and added Provider from Jotai and HydrateAtoms component. But there is an error with TypeScript in line 11: No overload matches this call.

However the code works fine in the browser.

![image](https://github.com/pmndrs/jotai/assets/69610657/f451a674-8a4c-4d08-9a95-e41622b33e22)

**Comment by dai-shi** [comment]
Thanks for reporting.
I just tried this, but doesn't show the error.
https://tsplay.dev/mb12bN
Can you modify the playground ☝️ to reproduce it? (Use "Link Shortener" plugin to create such a link.)

<img width="573" alt="image" src="https://github.com/pmndrs/jotai/assets/490574/33a71edc-3096-4bc6-940f-ba9d1b5015ec">

**Reply by JuanJoEstrada** [reply]
There was no error in the playground after I wrote the code. So I decided to hit "npm update" in my project and that solved the problem.

**Comment by zach-is-my-name** [comment]
The docs don't mention the need to install @tanstack/react-query only @tanstack/query-core.  This is the issue
